### PR TITLE
More bcftools args

### PR DESCRIPTION
**Pull request by aofarrel**
I successfully tested --print-header and it prints just the header as expected, but I don't know of a vcf that will let me test the other one.

This PR also contains the fixes in https://github.com/DataBiosphere/xvcfmerge/pull/32

### CODE REVIEW

File: vcf_merge.wdl [hunk 1]
```diff
@@ -14,7 +16,7 @@ task xVCFMerge {
     Int input_size = ceil(size(input_files, "GB"))
     Int final_disk_size = input_size + addl_disk_space
     runtime {
-        docker: "xbrianh/xsamtools:v0.5.2"
+        docker: "ucsctlp/xsamtools:v0.5.9"
```
Comment (jessebrennan) [inline vcf_merge.wdl:19]: Sorry it took so many versions to get the release right :sweat_smile:

**Review by jessebrennan — APPROVED** [review]
Looks good!

**Review by DailyDreaming — APPROVED** [review]

**Comment by aofarrel** [comment]
Alisa clarified that she wanted an implementation that allows for putting most of the arguments on the same line like what exists with xvcfview's view_options string, so I'm going to close this PR and try something else instead.

**Comment by aofarrel** [comment]
Nevermind, this is back on the menu as a temporary partial solution.

**Pull request merged**